src/Catalog/Persistence/AzureStorage.cs

```diff
@@ -295,48 +295,49 @@ protected override async Task<StorageContent> OnLoadAsync(Uri resourceUri, Cance
             await _throttle.WaitAsync();
             try
             {
-                string content;
-
-                using (var originalStream = new MemoryStream())
+                if (blob.Exists())
```

Comment:
Why did this change?

Reply:
This change looks unintented. Adding @scottbommarito who did this commit on this branch fd1ebbbb608d6bf89019677da161bb5697938b3b

I think I can revert the changes to this file as the throttling change went in via separate PR (https://github.com/NuGet/NuGet.Services.Metadata/pull/520/files), but want to get confirmation from Scott first.

Reply:
Yeah I think I must have accidentally reverted the changes to this file from https://github.com/NuGet/NuGet.Services.Metadata/pull/514 when pulling it into my branch.

Reply:
Fixed.